src/libcollections/btree/node.rs

```diff
@@ -43,17 +53,39 @@ use boxed::Box;
 const B: usize = 6;
 pub const CAPACITY: usize = 2 * B - 1;
 
+/// The underlying representation of leaf nodes. Note that is often unsafe to actually store
+/// these, since only the first `len` keys and values are assumed to initialized. See also
```

Comment:
"to be uninit" was a correction that you're missing "be" in that sentence, and I misparsed init and uninit.